### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by jocca1985** [AhmetHamzaEmra/Intelegent_Lock#7]
Just to share results of testing 
<img width="1094" alt="screen shot 2018-03-14 at 2 34 40 pm" src="https://user-images.githubusercontent.com/5808303/37405636-6fa2097a-2795-11e8-8e36-5705f1a9760c.png">

Do you maybe know about some open source data set? 
Also, did you play with some hyperparameters to increase accuracy?
At the end, I would appreciate if you send me the paper you used to implement this.

**Comment by jocca1985** [comment]
What is interesting is that I was using exactly this picture from phone for fine-tunning.
Also noticed that it stopped detecting/recognising after few restarts (just shows WARNING).

**Comment by AhmetHamzaEmra** [comment]
I could not find any open source dataset for liveness detection but if you know any that would be great. The reason you can pass the liveness detection is it check if it is more than %70 confident. you can increase that number and it will perform better.  Also, did you replace your photo for face recognition it seems like it is also not working? 
BTW every time you run the code it finetunes with the same data over and over which confuses and creates a bias model. I recommend you finetune once and delete the extra images and use the finetuned saved model.  Please let me know if somethings change 😄 
And also, the model is not based on a paper.

**Comment by jocca1985** [comment]
Thanks for your answer. Indeed it works better without fine-tuning. 
I will experiment with threshold definitely. 
Do you have an idea about some overall accuracy of the model (at least on some smaller dataset)? I know it's hard to collect data for such specific problem
